### PR TITLE
Refactor/swap

### PR DESCRIPTION
**Pull request by 0xthk**

### CODE REVIEW

File: src/components/PageHeader/WalletPopup/SwapLoading/index.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  position: relative;
+`
+
+const StyledCoinFace = styled.div`
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: #b5c0d0;
```
Comment (pyc92) [inline src/components/PageHeader/WalletPopup/SwapLoading/index.tsx:62]: get the color from theme config

File: src/components/PageHeader/WalletPopup/SwapLoading/index.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  &:nth-child(5) {
+    transform: translateZ(0.2em);
+  }
+`
+
+function SpinCoin({ src }: { src: string }) {
+  return (
+    <StyledCoin>
+      <StyledCoinFace>
+        <Image src={src} width={60} height={60} />
```
Comment (pyc92) [inline src/components/PageHeader/WalletPopup/SwapLoading/index.tsx:83]: no need to optimize the image in this use case. we can use `unoptimized` attribute to disable image optimization.

File: src/components/PageHeader/WalletPopup/SwapLoading/index.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  )
+}
+
+export default function SwapLoading({ swapState, fromTokenInfo, toTokenInfo, onClose, onSuccess }: Props) {
+  const { t } = useTranslation('', { keyPrefix: 'wallet_popup.swap' })
+  let title = ''
+  let desc = ''
+  let showCloseButton = false
+  let showSuccessButton = false
+  let body: React.ReactNode = <Image src={LoadingIndicator} unoptimized width="80px" height="80px" />
```
Comment (pyc92) [inline src/components/PageHeader/WalletPopup/SwapLoading/index.tsx:101]: add the `unoptimized` attribute to the element

**Review by pyc92 — CHANGES_REQUESTED** [review]

File: src/components/PageHeader/WalletPopup/SwapLoading/index.tsx (hunk 2 above)
Comment (0xthk) [reply src/components/PageHeader/WalletPopup/SwapLoading/index.tsx]: d26a8ae

File: src/components/PageHeader/WalletPopup/SwapLoading/index.tsx (hunk 1 above)
Comment (0xthk) [reply src/components/PageHeader/WalletPopup/SwapLoading/index.tsx]: d26a8ae

**Review by pyc92 — APPROVED** [review]

**Pull request merged**